test/internet/test-inspector-help-page.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+const common = require('../common');
+
+common.skipIfInspectorDisabled();
+
+if (!common.hasCrypto)
+  common.skip('missing crypto');
+
+const assert = require('assert');
+const https = require('https');
+const { spawnSync } = require('child_process');
+const child = spawnSync(process.execPath, ['--inspect', '-e', '""']);
+const stderr = child.stderr.toString();
+const helpUrl = stderr.match(/For help see (.+)/)[1];
```

Comment:
If https://github.com/nodejs/node/pull/19871 lands in its current form, this will need to be updated.